ddtrace/appsec/processor.py

```diff
@@ -87,6 +87,6 @@ def on_span_finish(self, span):
         if res is not None:
             # Partial DDAS-011-00
             log.debug("[DDAS-011-00] AppSec In-App WAF returned: %s", res)
-            span.meta["appsec.event"] = "true"
-            span.meta["_dd.appsec.json"] = '{"triggers":%s}' % (res,)
+            span.set_tag("appsec.event", "true")
```

Comment:
can we do _set_str_tag here for performance?

Reply:
Done